cadl-tests/generate.sh

```diff
@@ -9,10 +9,10 @@ function generate_remote {
         exit 1
     fi
 
-    cp -rf cadl-output/java/src/ .
```

Comment:
On mac, the extra slash will result in copy everthing under `src` but itself. 

Tested on both mac and windows.